src/models/VisualStyleModel/impl/cxVisualPropertyConverter.ts

```diff
@@ -305,6 +305,13 @@ export const VPNodeLabelPositionConverter = (
   }
 }
 
+export const VPPieSizeConverter = (): CXVisualPropertyConverter<string> => {
```

Comment:
This converter should just return the object you receive from the CX json for the CX vp 'NODE_CUSTOMGRAPHICS_1'